From 184c3db65594c6f216d5ef641f14705bc3aefbfd Mon Sep 17 00:00:00 2001
From: David Runge <dave@sleepmap.de>
Date: Mon, 27 Nov 2023 13:02:56 +0100
Subject: [PATCH 1/2] Use current appstream option for not accessing the
 network

In current appstreamcli the option for not accessing the network is
`--no-net`.
https://man.archlinux.org/man/appstreamcli.1#OPTIONS

Signed-off-by: David Runge <dave@sleepmap.de>
---
 data/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/meson.build b/data/meson.build
index 2e770d98..21621a47 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -62,7 +62,7 @@ if appstreamcli.found()
   test(
     'validate-appdata', appstreamcli,
     args: [
-      'validate', '--nonet', appdata_file.full_path()
+      'validate', '--no-net', appdata_file.full_path()
     ]
   )
 endif
-- 
2.43.0

